packages/components/mention/src/mention.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div :class="ns.b()">
+    <el-input
+      v-bind="mergeProps(passInputProps, $attrs)"
+      ref="elInputRef"
+      :model-value="modelValue"
+      @input="handleInputChange"
+      @keydown="handleInputKeyDown"
+      @focus="handleInputFocus"
+      @blur="handleInputBlur"
+      @mousedown="handleInputMouseDown"
+    >
+      <template v-for="(_, name) in $slots" #[name]="slotProps">
+        <slot :name="name" v-bind="slotProps" />
+      </template>
+    </el-input>
+    <el-tooltip
+      ref="tooltipRef"
+      :visible="visible && (!!filteredOptions.length || loading)"
+      :popper-class="[ns.e('popper'), popperClass]"
+      :popper-options="popperOptions"
+      :placement="`${placement}-start`"
+      :fallback-placements="['bottom-start', 'top-start']"
+      effect="light"
+      pure
+      :offset="0"
+      :show-arrow="false"
+    >
+      <template #default>
+        <div :style="cursorStyle" />
+      </template>
+      <template #content>
+        <el-mention-dropdown
+          ref="dropdownRef"
+          :options="filteredOptions"
+          :disabled="disabled"
+          :loading="loading"
+          @select="handleSelect"
+        >
+          <template v-for="(_, name) in $slots" #[name]="slotProps">
+            <slot :name="name" v-bind="slotProps" />
+          </template>
+        </el-mention-dropdown>
+      </template>
+    </el-tooltip>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { computed, mergeProps, nextTick, ref } from 'vue'
+import { pick } from 'lodash-unified'
+import { useNamespace } from '@element-plus/hooks'
+import ElInput, { inputProps } from '@element-plus/components/input'
+import ElTooltip from '@element-plus/components/tooltip'
+import { UPDATE_MODEL_EVENT } from '@element-plus/constants'
+import { isFunction } from '@element-plus/utils'
+import { mentionEmits, mentionProps } from './mention'
+import { getCursorPosition, getMentionCtx } from './helper'
+import ElMentionDropdown from './mention-dropdown.vue'
+
+import type { CSSProperties } from 'vue'
+import type { InputInstance } from '@element-plus/components/input'
+import type { TooltipInstance } from '@element-plus/components/tooltip'
+import type { MentionCtx, MentionOption } from './types'
+
+defineOptions({
+  name: 'ElMention',
+})
+
+const props = defineProps(mentionProps)
+const emit = defineEmits(mentionEmits)
+
+const passInputProps = computed(() => pick(props, Object.keys(inputProps)))
+
+const ns = useNamespace('mention')
+
+const elInputRef = ref<InputInstance | null>(null)
```

Comment:
These default values ​​of null should all be removed.
```suggestion
const elInputRef = ref<InputInstance>()
```

Reply:
done